src/main/java/hudson/scm/SubversionSCM.java

```diff
@@ -1667,8 +1667,10 @@ public void load() {
                 File jobCredentials = new File(job.getRootDir(), "subversion.credentials");
```

Comment:
I can't tell exactly how/why it happened, but I've found a few such jobs (with GitSCM in their `config.xml`, but still having a `subversion.credentials` file on-disk), on some of the Jenkins instances I manage.  My assumption was that it is some left-overs after projects moving from SVN to Git, and changing SCM config of their jobs.